applications/API/models.py

```
@@ -0,0 +1,733 @@
+        verbose_name_plural = _("Insult Categories")
+        managed = True
+        ordering = ["name"]
+        constraints = [
+            models.UniqueConstraint(
+                fields=["category_key", "name"], name="unique_category_key_name"
+            ),
+        ]
+        indexes = [
+            models.Index(fields=["category_key"], name="idx_category_key"),
+            models.Index(fields=["name"], name="idx_name"),
+        ]
```

Ruff (0.14.1)

Order of model's inner classes, methods, and fields does not follow the Django Style Guide: `Meta` class should come before Magic method

(DJ012)